interp/cache_test.go

```
@@ -3,23 +3,120 @@ package interp
+		require.True(t, ok)
+		require.Equal(t, first, request)
+		cache.fail(0)
+
+		request, ok = cache.claim(0, 1)
+		require.True(t, ok)
+		require.Equal(t, second, request)
+	})
+
+	t.Run("prioritizes side exits in arrival order", func(t *testing.T) {
+		cache := NewCache(program.New(nil))
+		defer cache.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `cache.Close` is not checked

(errcheck)